spec/services/essi/generate_pdf_service_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+
+RSpec.describe ESSI::GeneratePdfService do
+  let(:resource) { create(:paged_resource) }
+  let(:service) { described_class.new(resource) }
+  let(:pdf) { service.generate }
+
+  before do
+    pdf_fixtures = Rails.root.join('spec', 'fixtures', 'pdfs')
```

Comment:
these should be put in tmp if they are tmp files, not in spec/fixtures. then they don't need to be gitignored separately.